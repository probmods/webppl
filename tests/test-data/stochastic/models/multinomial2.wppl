var model = function() {
  return multinomial([0.4, 0.6], 2);
};
