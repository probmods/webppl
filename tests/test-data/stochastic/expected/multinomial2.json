{
  "hist": {
    "[2,0]": 0.16,
    "[1,1]": 0.48,
    "[0,2]": 0.36
  }
}
